config/honeybadger.php

```diff
@@ -128,4 +133,12 @@
             Breadcrumbs\ViewRendered::class,
         ],
     ],
+
+    /**
+     * Define your checkins here and synchronize them to Honeybadger with the
```

Comment:
This is helpful, you may want to include it in the README too. 

I looked at the docs you linked to and they could use a bit more information if we're allowing folks to set up their checkins like this instead of via the UI. Is there an issue for that?

Reply:
Yes, there an [issue](https://github.com/honeybadger-io/docs/issues/346) to update PHP/Laravel documentation. I will work on that asap!